### ISSUE TITLE
Possible to dual-license with Apache 2.0?

### ISSUE THREAD

**Issue by GBKS** [rsms/inter#338]
There is an effort to use Inter as the font for the [Bitcoin Core software](https://bitcoincore.org). However, there is strong pushback against the Open Font License (OFL), which seem to be centered on two aspects.

One is the fact that OFL is not irrevocable, meaning it could be revised. This is important to protect projects in the future. [See the discussion](https://github.com/bitcoin-core/gui/issues/89#issuecomment-744653910) the also shows how Apache 2.0 has that protection.

The second (and less important) argument is that fonts under OFL cannot be redistributed by themselves, but some sort of minimal addition has to be made. The logic is that this makes the font non-free. Not really an issue for Bitcoin Core, but it was brought up so I thought I'd mention it.

I assume that there is no intention to let licensing details get in the way of this great font family getting used. Would it be possible then do dual-license? That means adding the Apache 2.0 license and adding a note that anyone who uses this font can choose under which licenses they want to use it. It's basically an extra option for users.

I am also curious if there are considerations as to why not do do this.

**Comment by luke-jr** [comment]
> (and less important)... Not really an issue for Bitcoin Core,

Yes, it is absolutely a very important issue. We are not going to rely on non-free software.

**Comment by a4lg** [comment]
In general, OFL Frequently Asked Questions (OFL-FAQ) from https://software.sil.org/oflt/ says:

> 8.7 Does the OFL have an explicit expiration term?
> No, the implicit intent of the OFL is that the permissions granted are perpetual and irrevocable.

Besides, SIL-OFL is an OSI-approved license. How can it be approved without being irrevocable?

**Comment by KrasnayaPloshchad** [comment]
You need ask a lawyer for help.
